README.md

```diff
@@ -11,3 +11,154 @@ This Repository
 ===============
 
 This repository contains the markup needed to deploy the floeproject.org website.
+
+
+News Item Template
```

Comment:
@jhung I have instructions in there now - do you mean something different than what is there now?